app/src/main/java/cs5150athletetracking/com/athletetracking/AsyncUploader.java

```diff
@@ -0,0 +1,102 @@
+package cs5150athletetracking.com.athletetracking;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import java.io.*;
+import java.net.URL;
+import java.net.HttpURLConnection;
+import java.net.URLEncoder;
+import java.util.Iterator;
+
+import org.json.JSONObject;
+
+
+/**
+ * Created by mag94 on 10/19/15.
+ */
+
+public class AsyncUploader extends AsyncTask<JSONObject, String, Integer> implements ResultCallable {
```

Comment:
Yea I guess we could do that. Unless making callable an object that can get passed in as a parameter, in that case I would have to add a constructor. This might be better, unless you want to stick with interace